content/chainguard/chainguard-images/reference/php/getting-started.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Getting Started with the PHP Chainguard Images"
+type: "article"
+description: "Documentation on how to get started with the PHP Chainguard Images"
+date: 2023-01-09T11:07:52+02:00
+lastmod: 2023-01-09T11:07:52+02:00
+draft: false
+images: []
+referenceimage: php
+menu:
+  docs:
+    parent: "php-guides"
+weight: 10
+toc: true
+---
+
+The PHP images maintained by Chainguard are a mix of development and production distroless images that are suitable for building and running command-line PHP workloads.
+
+Because PHP applications typically require user-land package installs via Composer, using a pure distroless image for building your application would not work. In cases like this, you'll need to implement a [multi-stage Docker build](https://docs.docker.com/build/building/multi-stage/) that uses one of the `-dev` images to set up the application.
```

Comment:
"user-land" is a new word for me 😄

Reply:
I am going to change this 😄 it can be confusing, I am not sure everyone will understand it as I do, lol. but I meant dependencies that are not system-wide